### ISSUE TITLE
Can i use this tool via web application?

### ISSUE THREAD

**Issue by kjugk** [kyoshidajp/ec2spec#5]
Thank you for create such a  good tool 😄 

But i think how wonderful it would be if i can use this tool via web application.
Specifically, I wanna receive results by JSON below for example.

```
{
  host1: {
    instance_type: "t2.micro",
    instance_id: "i-xxxxxxxx",
    memory: "1016324kB",
  },
  ...
}
```

So, Are you planning to implement it?

**Comment by kyoshidajp** [comment]
@kjugk Thanks! I have already implemented by #6

**Comment by kjugk** [comment]
@kyoshidajp Great!! I'll try to create web app using this gem as soon as possible 👍
